pkg/filters/labels/filter.go

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright 2018 The Service Manager Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package labels
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/Peripli/service-manager/pkg/types"
+
+	"github.com/Peripli/service-manager/pkg/query"
+	"github.com/Peripli/service-manager/pkg/util"
+	"github.com/Peripli/service-manager/pkg/web"
+	"github.com/tidwall/gjson"
+)
+
+// ForbiddenLabelOperations describe denied label key operations
+type ForbiddenLabelOperations map[string][]query.LabelOperation
```

Comment:
what is the use case behind  allowing to create particular protected label but to disallow deleting a particular protected label  and vise versa ? It seems a bit odd to have a label that you can now have a label that can be deleted but cannot be added

Reply:
No particular use case, but in case it is needed in the future